website/docs/multiplatform-patching.md

```diff
@@ -2,134 +2,155 @@
 title: Multi-Platform Patching
 ---
 
-Copa also supports patching multi-platform container images, streamlining the process of securing applications deployed across diverse hardware platforms. This guide explains how Copa handles multi-platform images and how you can use this feature.
+This guide covers Copa's multi-platform patching capabilities for securing applications across diverse hardware platforms.
 
-## Usage
+## Overview
 
-Similar to patching single architecture images, Copa supports two ways to patch multi-platform container images:
+[Multi-platform images](https://docs.docker.com/build/building/multi-platform/) contain multiple platform-specific variants of the same application. Copa can automatically detect and patch these images across all supported platforms or target specific architectures based on your needs.
 
-1. **Report-based patching**: Use the `--report` flag pointing to a directory containing platform-specific vulnerability reports. This is the traditional approach that patches only platforms that exist and have respective reports.
+### Supported Architectures
 
-2. **Update all outdated packages**: When report flag is omitted, Copa detects if it is a manifest list and patches oudated packages across all platforms. Use the optional `--platform` flag to specify which platforms to patch.
+Copa supports patching the following platforms:
 
-### Method 1: Report-based Multi-Platform Patching
+| Platform        | Description                              |
+| --------------- | ---------------------------------------- |
+| `linux/amd64`   | 64-bit x86 (Intel/AMD)                   |
+| `linux/arm64`   | 64-bit ARM (Apple Silicon, AWS Graviton) |
+| `linux/arm/v7`  | 32-bit ARM v7                            |
+| `linux/arm/v6`  | 32-bit ARM v6                            |
+| `linux/386`     | 32-bit x86                               |
+| `linux/ppc64le` | PowerPC 64-bit Little Endian             |
+| `linux/s390x`   | IBM System z                             |
```

Comment:
should this also include `linux/riscv64`? we include it in the help in `pkg/patch/cmd.go`

Reply:
we didn't had this listed in `validPlatforms` @ashnamehrotra do we need to have `linux/riscv64` in https://github.com/project-copacetic/copacetic/blob/514b0f6b160559cd7191864f86123e49a6e688d8/pkg/patch/patch.go#L892

Reply:
it says not yet supported from the original reference used https://github.com/docker/cli/blob/74a896f18c9f8b10ce8a721f01c9f6fb71584987/cli/command/completion/functions.go#L154

so to be consistent I can remove it from cmd.go

Reply:
added `linux/riscv64`, since its a valid platform with #1199